R/mc5_mthds.R

```diff
@@ -22,18 +22,41 @@
 #' available in the package vignette, "Pipeline_Overview."
 #'
 #' \describe{
-#'   \item{bmad3}{Add a cutoff value of 3*bmad.}
-#'   \item{pc20}{Add a cutoff value of 20.}
-#'   \item{log2_1.2}{Add a cutoff value of log2(1.2).}
-#'   \item{log10_1.2}{Add a cutoff value of log10(1.2).}
-#'   \item{bmad5}{Add a cutoff value of 5*bmad.}
-#'   \item{bmad6}{Add a cutoff value of 6*bmad.}
-#'   \item{bmad10}{Add a cutoff value of 10*bmad.}
-#'   \item{log2_2}{Add a cutoff value of log2(2).}
-#'   \item{log10_2}{Add a cutoff value of log10(2).}
-#'   \item{neglog2_0.88}{Add a cutoff value of -1*log2(0.88).}
+#'   \item{bmad3}{Add a cutoff value of 3 multiplied by the baseline median absolute deviation 
+#'   (bmad). By default, bmad is calculated using test compound wells (wllt = t) for the endpoint.}
```

Comment:
This by default sentence only applies to single conc. Could replace as "Add a cutoff value of 3 multiplied by the baseline median absolute deviation (bmad) as defined at Level 4."